src/_includes/templates/forms/form-eeo-issue.html

```diff
@@ -0,0 +1,160 @@
+    <section class="block
+                    block__bg
+                    block__border
+                    block__flush-sides">
+        <form class="form-l js-validate-filters">
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="eeo_first-name">
```

Comment:
I worry that `eeo` isn't descriptive enough to id a form. Would `eeo-form` be better and then I can change all the ids to `eeo-form_`?